contrib/linux/sensors/file_watch_sensor.py

```diff
@@ -36,13 +33,25 @@ def cleanup(self):
                 pass
 
     def add_trigger(self, trigger):
-        pass
+        if trigger['type'] not in ['linux.file_watch.file_path']:
```

Comment:
I verified and we are filtering right in TriggerWatcher and sensors don't see triggers that they don't register.